chapters/traversal.md

```diff
@@ -1,83 +1,85 @@
 # Understand Jina Recursive Document Representation
```

Comment:
Please make sure all references to a document are with an uppercase D.

Reply:
oh, thanks for the hint, I will fix that right away